server/services/prisonerSearchService.test.ts

```diff
@@ -1,28 +1,21 @@
 import { Readable } from 'stream'
+import 'reflect-metadata'
```

Comment:
needed as this project uses `class-transformer` and does funky stuff with its DTOs (don't really know why)